src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,31 @@
+import cn from 'classnames';
+import { useContext } from 'react';
+import { FilterType } from '../../types/FilterType';
+import { FilterContext } from '../../contexts/FilterContext';
+
+export const Filter: React.FC = () => {
+  const { filter: filterType, setFilter } = useContext(FilterContext);
+
+  return (
+    <nav className="filter">
+      {Object.values(FilterType).map(filter => {
+        const isDefalutFilter = filter === FilterType.All;
+        const lowerFilterString = filter.toLowerCase();
+
+        return (
+          <a
+            key={filter}
+            href={`#/${isDefalutFilter
```

Comment:
move it to the variable
```suggestion
            href={...
```